packages/@best/frontend/src/modules/store/actions/actions.js

```diff
@@ -19,6 +21,31 @@ function normalizeCommit(commit) {
     return commit.slice(0, 7);
 }
 
+function shouldFetchOrganizations(state) {
```

Comment:
should add tests cases for the organization actions in `./__tests__`. See the projects-actions.spec.js for examples